### PR TITLE
added docs and corrected info code

### PR DESCRIPTION
**Pull request by sa-k-shore**
Feature Added :
https://github.com/DiceDB/dice/issues/146 Add support for COMMAND DOCS command
https://github.com/DiceDB/dice/issues/641 Add support for COMMAND DOCS command
Description

Add implementation of "command docs" (internal/eval/eval.go)
Add unit test for "command docs"
Add integration test for "command docs"
Add benchmark test for "command docs"

fixed inconsistency in "command info" (internal/eval/eval.go) wrt redis
slight modification of unit test for "command docs"
slight modification of  integration test for "command docs"

Problems Persisting :
we need to add many fields into CmdMeta like complexity etc, thanks

### CODE REVIEW

**Comment by sa-k-shore** [comment]
@arpitbbhayani  @JyotinderSingh  @lucifercr07

**Comment by lucifercr07** [comment]
@sa-k-shore please resolve conflicts

**Comment by sa-k-shore** [comment]
> @sa-k-shore please resolve conflicts

Done @lucifercr07

**Comment by apoorvyadav1111** [comment]
Hi @sa-k-shore, thank you for the changes. Please check any for any issues causing check failure. Additionally we have recent merges for Command subcommands docs. Please rebase your branch to get the latest changes.
Thanks

**Comment by apoorvyadav1111** [comment]
Hi @sa-k-shore, Hope you are well. I am excited to know about your progress on this issue. As you know, we would like to close this issue soon, please reach out to me or anyone on the discord community if you need any help.

Thanks.

**Comment by sa-k-shore** [comment]
Screenshot 2024-10-21 at 12 20 25 AM
hi @apoorvyadav1111  can i know why are this debug logs coming as errors are they errors ?

**Comment by sa-k-shore** [comment]
@apoorvyadav1111  sorry for delay , can you rerun the workflows if they fail i will fix them

**Comment by sa-k-shore** [comment]
can you re run it @apoorvyadav1111

**Comment by sa-k-shore** [comment]
hi @apoorvyadav1111  can you rerun workflow, thanks

**Review by lucifercr07 — APPROVED** [review]
LGTM

File: integration_tests/commands/async/command_info_test.go
Comment (lucifercr07) [inline integration_tests/commands/async/command_info_test.go:1]: Shall we add websocket integration tests also?

**Comment by lucifercr07** [comment]
@sa-k-shore please resolve conflicts and check this comment once. https://github.com/DiceDB/dice/pull/993#discussion_r1810586699

**Comment by sa-k-shore** [comment]
hi @lucifercr07  its been long since this pr is not merged can we merge it and i will write those its as another pr and will get it merged.

**Comment by lucifercr07** [comment]
> hi @lucifercr07 its been long since this pr is not merged can we merge it and i will write those its as another pr and will get it merged.

Okay please go ahead and create an issue for the same, will assign, then we can proceed with merge.

**Comment by sa-k-shore** [comment]
created one issue can you assign that to me #1192

**Pull request merged**